src/renderer.zig

```diff
@@ -166,6 +168,29 @@ pub fn render(playerPosition: Vec3d) void {
 	}
 }
 
+pub fn crosshairDirection(rotationMatrix: Mat4f, fovY: f32, width: u31, height: u31) Vec3f {
+	// stolen code from Frustum.init
+	const invRotationMatrix = rotationMatrix.transpose();
+	const cameraDir = vec.xyz(invRotationMatrix.mulVec(Vec4f{0, 0, 1, 1}));
+	const cameraUp = vec.xyz(invRotationMatrix.mulVec(Vec4f{0, 1, 0, 1}));
```

Comment:
It looks like this code still comes from the y-up times.
`Frustum` isn't actually used anymore, and it seems that its code silently broke in the transition to z-up.